apps/emqx_bridge_rocketmq/src/emqx_bridge_rocketmq_connector.erl

```diff
@@ -317,29 +316,29 @@ acl_info(_, _, _) ->
     #{}.
 
 create_producers_map(ClientId) ->
-    erlang:spawn(fun() ->
-        case ets:whereis(ClientId) of
-            undefined ->
-                _ = ets:new(ClientId, [public, named_table]),
-                ok;
-            _ ->
-                ok
-        end,
-        receive
-            _Msg ->
-                ok
-        end
-    end).
-
-get_producers(ClientId, {_, Topic1} = TopicKey, ProducerOpts) ->
-    case ets:lookup(ClientId, TopicKey) of
-        [{_, Producers0}] ->
-            Producers0;
+    _ = ets:new(ClientId, [public, named_table, {read_concurrency, true}]),
```

Comment:
we can now avoid crating this ets table?

the allocated resource table can be used to lookup producers.

Reply:
No, I have tried. The resource table is a bag which unsuited for the RocketMQ, because the `topic` in the RocketMQ bridge should support the placeholders, so the amount of `topic` may be huge, if we insert all of the `topic` into the resource table, each time when we lookup a producer, we have to fall into a long list